### DISCUSSION TITLE
Welcome to WebDeck Discussions!

### DISCUSSION THREAD

**Discussion by Lenochxd** [Lenochxd/WebDeck#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Sam-U-ElJackson** [comment]
Can WebDeck be used remotely or does it need to be on the same network. As an IRL streamer it would great to find an alteritive for the Streamdeck that can be IP connected outside your home network

**Reply by Lenochxd** [reply]
Hey, I'm not sure if this is possible in the latest version, but it will be in the next release :)
